### PR TITLE
Change indexing order of nested loops

### LINKED ISSUE #52: Modify 04. Loops.ipynb in order to reflect "correct" indexing order
I think that https://github.com/JuliaComputing/JuliaBoxTutorials/blob/master/introductory-tutorials/intro-to-julia/04.%20Loops.ipynb should be adjusted to reverse the indexing order in the section *for loops*.

Maybe a brief note to the performance reasons behind this and a link to https://docs.julialang.org/en/v1/manual/performance-tips/#Access-arrays-in-memory-order,-along-columns-1 would be nice as well.

### PR DESCRIPTION
**Pull request by julian-deltares**
Closes #52

### CODE REVIEW

**Comment by StefanKarpinski** [comment]
Since it's quite conventional for the indices to be in `i, j` order, I think it would be preferable to change the order of the loops.

**Comment by julian-deltares** [comment]
> 
> 
> Since it's quite conventional for the indices to be in `i, j` order, I think it would be preferable to change the order of the loops.

Good point.
I adjusted it to according to your suggestion.

**Comment by StefanKarpinski** [comment]
LGTM. @mbauman, are you the right person to approve this?

File: introductory-tutorials/intro-to-julia/04. Loops.ipynb [hunk 1]
```diff
@@ -103,6 +103,8 @@
    "source": [
     "Now let's use `for` loops to create some addition tables, where the value of every entry is the sum of its row and column indices. <br>\n",
     "\n",
+    "For more information about fast indexing of multidimensional arrays inside nested loops can be found at https://docs.julialang.org/en/v1/manual/performance-tips/#Access-arrays-in-memory-order,-along-columns-1",
```
Comment (NHDaly) [inline introductory-tutorials/intro-to-julia/04. Loops.ipynb]:
```suggestion
    "Note that we iterate over this array via column-major loops in order to get the best performance. More information about fast indexing of multidimensional arrays inside nested loops can be found at https://docs.julialang.org/en/v1/manual/performance-tips/#Access-arrays-in-memory-order,-along-columns-1",
```

Comment (xorJane) [reply introductory-tutorials/intro-to-julia/04. Loops.ipynb]: LGTM as well, and I like your suggestion Nathan!

Comment (NHDaly) [reply introductory-tutorials/intro-to-julia/04. Loops.ipynb]: Applied manually in e61c7f9.

**Pull request merged**